packages/blade/src/components/Amount/Amount.tsx

```diff
@@ -99,11 +106,11 @@ const AmountValue = ({
     const integer = value.split('.')[0];
     const decimal = value.split('.')[1];
 
-    // Native does not support alignItems of Text inside a div, insted we need to wrap is in a Text
-    const AmountWrapper = getPlatformType() === 'react-native' ? Text : React.Fragment;
+    // Native does not support alignItems of Text inside a div, instead we need to wrap is in a Text
+    const AmountWrapper = isReactNative ? BaseText : BaseBox;
 
     return (
-      <AmountWrapper>
+      <AmountWrapper position="relative">
```

Comment:
Amount is a Typography component. Can we avoid adding a div to it?

Reply:
This is needed for the strikethrough. Maybe we can change it to `Text` here, but we need at least one `div` below for the strike-through.